.github/actions/build-and-test/action.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Builds and Test'
+description: 'Builds the library and runs the unit tests'
+
+runs:
+  using: composite
+
+  steps:
+
+    - name: Restore Packages
+      shell: bash
```

Comment:
Are you able to specify this as a default? Also is there a partic reason the be explicit about it here?

Reply:
It's required for some reason. I tried taking shell as in input but that doesn't work either; actually found an [issue](https://github.com/actions/runner/issues/835) about it.